templates/partials/summary/list-summary.html

```diff
@@ -1,38 +1,42 @@
 {% from "components/summary/_macro.njk" import onsSummary %}
 
-{% if list.list_items | length > 0 %}
-
-  {% if list.editable %}
-    {% set headers = ["Name of person", "Action"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person",
-      _("Change"),
-      _("Change details for {item_name}"),
-      _("Remove"),
-      _("Remove {item_name}")
-    ) %}
-  {% else %}
-    {% set headers = ["Name of person"] %}
-    {% set rows = map_list_collector_config(
-      list.list_items,
-      "person"
-    ) %}
-  {% endif %}
+{% if list.editable %}
+  {% set headers = ["Name of person", "Action"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person",
+    _("Change"),
+    _("Change details for {item_name}"),
+    _("Remove"),
+    _("Remove {item_name}")
+  ) %}
+{% else %}
+  {% set headers = ["Name of person"] %}
+  {% set rows = map_list_collector_config(
+    list.list_items,
+    "person"
+  ) %}
+{% endif %}
 
-  {{ onsSummary({
-    "withinQuestion": true,
-    "summaries": [
-      {
-        "groups": [
-          {
-            "headers": headers,
-            "rows": rows,
-          }
-        ]
-      }
-    ],
-    "classes": "u-mt-s u-mb-m"
-  }) }}
+{% set groups_config = {
+  "groupTitle": summary_title,
```

Comment:
I think in context of this file, we prob should update this to just `title` or `group_title`.

Reply:
Changed it now, thanks!